Makefile.dev

```diff
@@ -78,7 +107,7 @@ build-image-ti:
 	  --kubeconfig $(KUBECONFIG) \
 	  -n $(NAMESPACE) \
 	  --name $(NAME) \
-	  --manifest $(MANIFEST_FILE) \
+	  --manifest $(OUTPUT_FILE) \
```

Comment:
@bennyz please note this, it's pointing now to the output of the manifest change